src/website/app/demos/Form/index.js

```diff
@@ -46,7 +46,19 @@ export default [
     slug: 'form-field',
     title: 'FormField',
     whenHowToUse: `Wrap each input in your app with a FormField for appropriate
-accessibility and styling.`
+accessibility and styling.
+
+Labels should be short (1-3 words) and indicate the type of input requested.
+Labels are not meant to explain the intent of the form as a whole.
+
+Only use [Sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) for labels.
+
+Placeholder texts are example input values, not instructions, as the contrast is too low to be accessible.
+
+Do not use an asterisk in labels for required fields; use the \`required\` attribute instead.
+
+Error text should be short and concise, telling the user how to fix the problem.
+Form errors should be shown next to the pertinent input field – as opposed to the top of the form – so the user knows exactly what to change.`
```

Comment:
Regarding form errors being displayed next to the form field and not at the top of the form...  I think that this is a bit too prescriptive.  A form level list of validation errors can also be done in an accessible way.  It might likely even improve the accessibility of individual field error messages.  Showing these individual field errors are only fully accessible if the field is given focus following the error.  A top level notification/alert could list the fields with problems and provide links to focus the individual fields.

I would suggest to remove this line for now as it seems beyond the scope of this story.

Reply:
removing for now. We should circle the wagons and have a recommendation for this sort of thing, just so people don't do it the wrong way.